service/history/api/signalwithstartworkflow/signal_with_start_workflow.go

```diff
@@ -186,6 +186,8 @@ func createWorkflowMutationFunction(
 		workflowIDReusePolicy,
 		workflowIDConflictPolicy,
 		currentWorkflowStartTime,
+		nil,
+		false,
```

Comment:
Maybe keeping the flag in `ParentExecutionInfo` would have been better? Now we have to pass 2 params all the way down.

Reply:
if ChildWorkflowOnly is false then we don't need to pass parentExecutionInfo down...  so for ResolveDuplicateWorkflowID we can say as long as parentExecutionInfo is specified, we will verify the relationship. That could be confusing for the caller though... 

I am ok with passing both down. I didn't know we have that many layers though 🤦 

I still feel it's a bit weird for it to be in `ParentExecutionInfo` because the info is stored in the child, but the verification and termination is done on a different workflow.

Reply:
Agree. Passing both is the best possible option.